python/nav/web/sortedstats/views.py

```diff
@@ -82,8 +82,13 @@ def process_form(form):
     rows = form.cleaned_data['rows']
     cache_key = get_cache_key(view, timeframe, rows)
     if form.cleaned_data['use_cache']:
-        result = get_cache().get(cache_key)
-        if result and not result.data:
+        try:
+            cache = get_cache()
+            result = cache.get(cache_key)
+            if result and not result.data:
+                result = None
+        except InvalidCacheBackendError as e:
+            _logger.error("Error accessing cache for ranked statistics: %s".format(e))
```

Comment:
This log statement appears to use some inconsistent string formatting techniques. As always, when using loggers, I suggest using the built-in string formatting of the logger object, which ensures that time is spent to format the arguments *only if the current log level dictates that this log record should be emitted*:

```suggestion
            _logger.error("Error accessing cache for ranked statistics: %s", e)
```

Reply:
fixed